stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/models/PinnedMessagesPagination.kt

```diff
@@ -0,0 +1,55 @@
+package io.getstream.chat.android.client.api.models
+
+import java.util.Date
+
+/**
+ * Pagination options for getting pinned messages.
+ */
+public sealed class PinnedMessagesPagination {
+
+    /**
+     * Pagination option for getting pinned messages around the message with given id.
```

Comment:
To maybe make this a bit more concise, maybe we can just start these sentences different.

Instead of: "Pagination option for getting..." We could say:

"Returns messages around the message with given id."

"Returns messages before the message with given id."

"Returns messages around the date."

...